### PR TITLE
Mashraf/about page new design

### PR DESCRIPTION
**Pull request by 22mahmoud**

### CODE REVIEW

File: src/pages/AboutPage/index.tsx [hunk 1]
```diff
@@ -34,353 +44,141 @@ class About extends Component<IProps, IState> {
   state = {
     data: {},
   };
+
   async componentDidMount() {
-    // Get the data from the backend.
     const data = humps.camelizeKeys(await fetchAboutData());
```
Comment (piraka9011) [inline src/pages/AboutPage/index.tsx]: Can we have a boolean set to true when `data` is empty then we don't load the graphs and load an error message instead? 
Maybe something simple like:
```
import _ from "lodash";
const isDataAvailable = _.isEmpty(data);

{ isDataAvailable : ? (
  <AyahsContributedCounter
     recitedAyahs={this.state.data.recordingCount}
     recordingCount={this.props.profile.recordingCount}
  /> 
  ) : (
  <p> Servers are not available right now! Please email bugs@tarteel.io </p>
)}
```

File: src/pages/AboutPage/Team/data.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  nameKey: KEYS;
+  positionKey: KEYS;
+  linkedinURL: string;
+  facebookURL: string;
+  imageSrc: string;
+}
+
+const teamData: ITeam[] = [
+  {
+    nameKey: KEYS.CONTRIBUTOR_NAME_1,
```
Comment (piraka9011) [inline src/pages/AboutPage/Team/data.ts]: Always Number 1 ;)

File: src/locale/en.ts [hunk 3]
```diff
@@ -154,25 +154,29 @@ const EN: IShape = {
...
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TITLE]:
+      'AI for Perfecting Quran Recitation',
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TEXT_1]:
+      'Tarteel is a Quran application that listens to your recitation and corrects mistakes. The name tarteel comes from the Quran itself, where God commands us to "recite the Quran with Tarteel (slow, measured, rhythmic tones)" (73:4).',
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TEXT_2]:
+      'Tarteel leverages the latest technology in AI and Machine Learning (ML) to perform speech recognition, and analyze recitations. To power our ML algorithms, we require a large and diverse dataset of reciters from all around the world to contribute their recitations. The more submissions the algorithm gets, the more accurate it becomes.',
```
Comment (piraka9011) [inline src/locale/en.ts]: Remove comma:
> perform speech recognition and analyze recitations

File: src/locale/ar.ts [hunk 4]
```diff
@@ -103,21 +103,34 @@ export default {
...
+      'بحمد الله, ساعد مستخدمي ترتيل <a href="/contribute">بالمساهمة</a> حتي الان للوصول الي اجمالي <b>{recitedAyahs}</b> اية',
     [KEYS.ABOUT_PAGE_TEMPLATE_TITLE]: 'عن ترتيل',
-    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TITLE]: 'ما هو ترتيل ؟',
-    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TEXT]:
-      'ترتيل هو مشروع مفتوح المصدر صُمم للمساعده في بناء برامج لتحليل قراءة القرآن. بالنظر لأهمية تلاوة القرآن في حياة المسلمين ، من المهم بناء أدوات برمجية يمكن أن تساعد المسلمين العاديين على قراءة القرآن بشكل أكثر دقة. اسم ترتيل يأتي من القرآن نفسه ، حيث يأمرنا الله بتلاوة القرآن مع الترتيل (أَوْ زِدْ عَلَيْهِ وَرَتِّلِ الْقُرْآنَ تَرْتِيلا)(73:4).',
-    [KEYS.ABOUT_PAGE_SECOND_PARAGRAPH_TITLE]: 'ما هو تحدي ترتيل 100,000 ؟',
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TITLE]:
+      'الذكاء الاصطناعي لاتقان تلاوة القرآن',
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TEXT_1]:
+      'ترتيل هو تطبيق للقرأن الكريم يقوم بالأستماع إلي تلاوتك و تصحيح الاخطاء.  اسم ترتيل يأتي من القرآن نفسه ، حيث يأمرنا الله بتلاوة القرآن مع الترتيل (أَوْ زِدْ عَلَيْهِ وَرَتِّلِ الْقُرْآنَ تَرْتِيلا)(73:4).',
```
Comment (piraka9011) [inline src/locale/ar.ts]: Add: سبحانه وتعالى
>يأمرنا الله

File: src/locale/ar.ts [hunk 5]
```diff
@@ -103,21 +103,34 @@ export default {
...
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TEXT_2]:
+      'تستفيد ترتيل من احدث التقنيات فى مجال الذكاء الاصطناعي والتعلم الآلي. لتتعرف علي الكلام وتحليل التلاوات. حتي تعمل الخوارزميات الخاصة بنا بكفاءة, نحتاج إلي مجموعة كبيرة ومتنوعة من القراء من جميع أنحاء العالم للمساهمة بتلاواتهم. فكلما زاد عدد المساهمات, اصبحت الخوارزميات اكثر دقة.',
```
Comment (piraka9011) [inline src/locale/ar.ts]: على
> علي

برامج الذكاء الاصطناعي

> الخوارزميات

File: src/locale/ar.ts [hunk 6]
```diff
@@ -103,21 +103,34 @@ export default {
...
     [KEYS.ABOUT_PAGE_SECOND_PARAGRAPH_TEXT_2_LINK]: 'مجموعة واسعة من التطبيقات',
-    [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TITLE]: 'فريق عمل ترتيل',
+    [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TITLE]: 'التقدم المحرز حتى الآن',
     [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TEXT]: 'تم إنشاء ترتيل بإذن الله ثم جهود:',
-    [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TITLE]: 'كيف يمكنني أن أقدم المساعدة؟',
+    [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TITLE]:
+      'هل الآيات التي يتم تلاوتها متنوعة؟',
     [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TEXT]:
-      "ترتيل هو مشروع مفتوح المصدر يتم تطويره من قبل المطورين المهتمين بالعمل علي ترتيل .إذا كنت ترغب في المساهمة ، يرجي مراجعة : <a href='https://github.com/Tarteel-io/tarteel.io'>https://github.com/Tarteel-io/tarteel.io</a>",
+      'يساعدنا لبناء نماذج جيدة للتعلم الآلي (ML) ان نجعل الناس تتلو آيات مختلفة من القرآن. نحن نحصي مدي تغطيتنا للقرآن كما مبين في  الشكل البياني التالي:',
```
Comment (piraka9011) [inline src/locale/ar.ts]: Remove ML

File: src/locale/ar.ts [hunk 7]
```diff
@@ -209,18 +222,38 @@ export default {
...
+      'طالبة علوم الحاسوب في معهد ماساتشوستس للتكنولوجيا',
+    [KEYS.CONTRIBUTOR_NAME_11]: 'محمود اشرف',
+    [KEYS.CONTRIBUTOR_POSITION_11]: 'مهندس برمجيات في ترتيل',
+    [KEYS.CONTRIBUTOR_NAME_12]: 'محمد موسى',
+    [KEYS.CONTRIBUTOR_POSITION_12]: 'مهندس برمجيات في فيسبوك',
+    [KEYS.CONTRIBUTOR_NAME_13]: 'مؤمن سليمان',
+    [KEYS.CONTRIBUTOR_POSITION_13]: 'مهندس برمجيات في سايبر تالنتس',
+    [KEYS.CONTRIBUTOR_NAME_14]: 'عمرو الفاس',
+    [KEYS.CONTRIBUTOR_POSITION_14]:
+      'عضو مجلس إدارة الجمعية الإسلامية البريطانية',
```
Comment (piraka9011) [inline src/locale/ar.ts]: في بوسطن
> البريطانية

File: src/pages/AboutPage/AyahsContributedCounter/index.tsx [hunk 8]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```
Comment (Ananasship) [inline src/pages/AboutPage/AyahsContributedCounter/index.tsx]: Nice!

File: src/pages/AboutPage/Team/index.tsx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+  intl: InjectedIntl;
+}
+
+function Team({ intl }: IProps) {
+  const rtl = intl.messages.local === 'arabic' ? 'rtl' : '';
+
+  return (
+    <React.Fragment>
+      <h2 className={rtl}>
+        <T id={KEYS.ABOUT_PAGE_FIFTH_PARAGRAPH_TITLE} />
```
Comment (Ananasship) [inline src/pages/AboutPage/Team/index.tsx]: Nit: can we add a little bit of space under the title.

File: src/locale/en.ts [hunk 10]
```diff
@@ -154,25 +154,29 @@ const EN: IShape = {
...
+    [KEYS.ABOUT_PAGE_FIRST_PARAGRAPH_TEXT_3]:
+      'Our goal is to support and serve our Imams and Institutions that seek to teach Muslims how to read the words of Allah - SWT - and to provide those with limited access to Imams and Sheikhs a means of maintaining their relationship with Allah - SWT - and his book.',
     [KEYS.ABOUT_PAGE_SECOND_PARAGRAPH_TITLE]:
-      'What is the Tarteel 100,000 challenge?',
+      'Why are you collecting recitations?',
```
Comment (Ananasship) [inline src/locale/en.ts]: Nit: can we change this to `Why are you we collecting recitations?`

Thanks Mahmoud

File: src/locale/en.ts [hunk 11]
```diff
@@ -154,25 +154,29 @@ const EN: IShape = {
...
-    [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TITLE]: 'How can I help?',
+    [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TITLE]: 'Progress to date',
+    [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TITLE]:
+      'Are the verses that are being recited varied?',
     [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TEXT]:
-      "Tarteel is an open-source project that is maintained by developers interested in furthering the mission of Tarteel. If you'd like to contribute, please <a href='contact.tarteel@gmail.com'>email us</a> or check out our GitHub repos: <a href='https://github.com/Tarteel-io/'>https://github.com/Tarteel-io/</a>",
+      "To build good machine learning models, it helps to have people recite different verses from across the entire Quran. We're measuring how well we've covered the Qur'an and these are our stats so far:",
     [KEYS.ABOUT_PAGE_FIFTH_PARAGRAPH_TITLE]:
-      'What is the demographic breakdown of Tarteel users?',
+      'The Team',
```
Comment (Ananasship) [inline src/locale/en.ts]: Nit: The Tarteel Team

File: src/locale/en.ts [hunk 12]
```diff
@@ -154,25 +154,29 @@ const EN: IShape = {
...
     [KEYS.ABOUT_PAGE_SECOND_PARAGRAPH_TEXT_2_LINK]:
       'a wide variety of applications',
-    [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TITLE]: 'Who is a part of Tarteel?',
-    [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TEXT]:
-      "Tarteel came out bi'iznillah and with the efforts of the following people:",
-    [KEYS.ABOUT_PAGE_FOURTH_PARAGRAPH_TITLE]: 'How can I help?',
+    [KEYS.ABOUT_PAGE_THIRD_PARAGRAPH_TITLE]: 'Progress to date',
```
Comment (Ananasship) [inline src/locale/en.ts]: Nit: Our progress to date

File: src/pages/AboutPage/styles.ts [hunk 13]
```diff
@@ -3,80 +3,64 @@ import styled from 'styled-components';
...
+  @media screen and (max-width: ${props => props.theme.breakpoints.sm}px) {
+    padding: 10px;
+    overflow-x: hidden;
   }
 
-  .content {
-    padding: 1em 0;
-    width: 75%;
-    margin: auto;
+  footer {
```
Comment (Ananasship) [inline src/pages/AboutPage/styles.ts]: Not a change here, but it looks like our contact us button is broken. Can we redirect it to the /contact page

File: src/locale/ar.ts (hunk 5 above)
Comment (Ananasship) [inline src/locale/ar.ts]: yastafeed

File: src/locale/ar.ts (hunk 6 above)
Comment (Ananasship) [reply src/locale/ar.ts]: I was actually going to say include it in other places as well 😂 and spell it out as Machine Learning. No strong opinions

Comment (piraka9011) [reply src/locale/ar.ts]: After discussing in person I agree, lets expand it to "Machine Learning" since it's not widely used Arabic terminology :)

**Comment by piraka9011** [comment]
[About Page](https://trello.com/c/eKPAw9j8/4-about-page)

**Pull request merged**